### PR TITLE
Log bridge metrics

### PR DESCRIPTION
**Pull request by rudle**
logbridge needs a metric registry so we can understand log lines per unit time and log bytes per unit time.

### CODE REVIEW

File: pkg/logbridge/logbridge.go [hunk 1]
```diff
@@ -6,50 +6,80 @@ import (
...
+type MetricsRegistry interface {
+	Register(metricName string, metric interface{}) error
+}
+
+func NewLogBridge(r io.Reader,
+	durableWriter io.Writer,
+	lossyWriter io.Writer,
+	logger logging.Logger,
+	metricsRegistry MetricsRegistry,
+	loglineMetricKeyName string,
```
Comment (anthonybishopric) [inline pkg/logbridge/logbridge.go]: We should standardize on a way to name metrics, as the convention elsewhere in this repo is to name the keys whatever we please when the registry is passed to us (ie they are constants). Check consul_aggregator.go for an example.

I'm not married by any means to that approach, in fact I'd almost prefer a central location for metric names - lets dedicate a few minutes of time to figuring that out.

Comment (anthonybishopric) [inline pkg/logbridge/logbridge.go]: Also not important that it's sorted in this PR per se, but I'd like to track it at the very least.

Comment (rudle) [inline pkg/logbridge/logbridge.go]: my primary concern about injecting here was not muddling up the constructor with: podID, az, clusterName and host. I agree that centralizing them would be good - but I think such belongs on the Square side of things.

Comment (anthonybishopric) [inline pkg/logbridge/logbridge.go]: There is probably a reasonable middle ground in letting this project define metric suffixes and letting the Square side provide a single prefix - that would get us the best of both worlds presumably

Comment (rudle) [inline pkg/logbridge/logbridge.go]: I don't know that we want a "single prefix" for all of Square. I would say that we want a single prefix for all pod-wise metrics like this one and another single prefix for all (for example) datastore watch metrics.

I think that we're agreeing, but I want to clarify the goal.

Comment (mpuncel) [inline pkg/logbridge/logbridge.go]: i agree we need to standardize metrics names somehow. but doesn't have to block this PR

**Comment by mpuncel** [comment]
👍

**Comment by rudle** [comment]
Thanks for the review!

**Pull request merged**